lib/scala/project-manager/src/main/resources/application.conf

```diff
@@ -73,12 +79,22 @@ logging-service {
           immediate-flush = true
         },
         {
-          name = "sentry"
-          dsn = "<dsn>"
-          dsn = ${?ENSO_APPENDER_SENTRY_DSN}
+          name = "opensearch",
+          uri = "https://api.ensoanalytics.com"
```

Comment:
Is this a correct production URI?

Reply:
Removing hardcoded URLs is tracked in https://github.com/enso-org/enso/issues/12564

Reply:
Yes, might as well do it partially here. This requires support from cloud, however.